website/docs/d/is_reservations.html.markdown

```diff
@@ -0,0 +1,151 @@
+    - `status` - (String) The status of the capacity reservation.
+
+      ->**status** 
+        </br>&#x2022; allocating: The capacity reservation is being allocated for use
+        </br>&#x2022; allocated: The total capacity of the reservation has been allocated for use
+        </br>&#x2022; degraded: The capacity reservation has been allocated for use, but some of the capacity is not available
+        </br>&#x2022; unallocated: The capacity reservation is not allocated for use
+    - `total` - (Integer) The total amount of this capacity reservation.
+    - `used` - (Integer) The amount of this capacity reservation used by existing attachments.
+  - `committed_use` - (List) The committed use configuration for this reservation. If absent, this reservation has no commitment for use.
+
+    Nested scheme for `committed_use`:
+    - `expiration_at` - (Timestamp) The expiration date and time for this committed use reservation.
+    - `expiration_policy` - (String) The policy to apply when the committed use term expires.
+
+      ->**expiration_policy** 
+        </br>&#x2022; release: Release any available capacity and let the reservation expire
+        </br>&#x2022; renew: Renew for another term, provided the term remains listed in the reservation_terms for the profile. Otherwise, let the reservation expire
+    - `term` - (String) The term for this committed use reservation.
+
+      ->**term** 
+        </br>&#x2022; one_year: 1 year
+        </br>&#x2022; three_year: 3 years
+  - `created_at` - (Timestamp) The date and time that the reservation was created.
+  - `crn` - (String) The CRN for this reservation.
+  - `href` - (String) The URL for this reservation.
+  - `id` - (String) The unique identifier for this reservation.
+  - `lifecycle_state` - (String) The lifecycle state of this reservation.
+
+    ->**lifecycle_state** 
+        </br>&#x2022; deleting
+        </br>&#x2022; failed
+        </br>&#x2022; pending
+        </br>&#x2022; stable
+        </br>&#x2022; suspended
+        </br>&#x2022; updating
+        </br>&#x2022; waiting
+  - `profile` - (List) The virtual server instance profile this reservation. 
+
+    Nested scheme for `profile`:
+    - `href` - (String) The URL for this virtual server instance profile.
+    - `name` - (String) The globally unique name for this virtual server instance profile.
+    - `resource_type` - (string) The resource type
+
+      ->**resource_type** 
+        </br>&#x2022; instance_profile
+  - `resource_group` - (List) The resource group for this reservation. 
+
+    Nested scheme for `resource_group`:
+    - `href` - (String) The URL for this resource group.
+    - `id` - (String) The unique identifier for this resource group.
+    - `name` - (String) The name for this resource group.
+  - `resource_type` - (String) The resource type.
+
+    ->**resource_type** 
+      </br>&#x2022; reservation
+  - `status` - (String) The status of the reservation.
+
+    ->**status** 
```

Comment:
Can you check other doc and see that we are mentioning allowed values this way.. i feel it makes the doc very lengthy..

Reply:
Yes, I referred to other is_* doc(s) and created this.